src/controllers/userGroupController.js

```diff
@@ -0,0 +1,22 @@
+const sequelize = require('../data-access/database');
+
+export default class UserGroup {
+    constructor(usersList, groupList) {
+        this.usersList = usersList;
+        this.groupList = groupList;
+    }
+
+    async addUserToGroup(req, res) {
+        await sequelize.sync();
+        await sequelize.transaction(async (t) => {
+            const user = await this.usersList.findOne({ where: { id: req.query.userId } });
```

Comment:
@lenon777, you should not search users and groups before adding them to `userGroup` table, because they should have foreign keys.